src/eventbrite-provider.ts

```diff
@@ -1,5 +1,7 @@
 /* Copyright © 2021 Seneca Project Contributors, MIT License. */
 
+import Eventbrite from 'eventbrite';
```

Comment:
This code uses a relatively standard format, but not semi-colons. Stay consistent with the existing code base as a general PR tactic.